test/run-system-test.sh

```diff
@@ -17,44 +17,56 @@ BIN_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 PACKAGE_DIR="${BIN_DIR%/*}"
 cd "${PACKAGE_DIR}/test"
 
-docker compose --env-file .env -p system-test down --volumes
-#
-# Note: cannot run as single docker compose command due to different build contexts
-docker compose --env-file .env -p system-test up --wait -d --build --remove-orphans
-# Warning: Requires to be run from the project root
-(cd .. && \
-  docker compose --env-file test/.env -p system-test up --wait -d --build)
-
-./scripts/insert_test_data.sh
-
-# Install pixl_cli and test dependencies
-pip install -e "${PACKAGE_DIR}/pixl_core" && \
-  pip install -e "${PACKAGE_DIR}/cli" && \
-  pip install -e "${PACKAGE_DIR}/pytest-pixl"
-
-
-pixl populate "${PACKAGE_DIR}/test/resources/omop"
-pixl start
-
-# need to wait until the DICOM image is "stable" so poll for 2 minutes to check
-./scripts/check_entry_in_orthanc_anon_for_2_min.py
-./scripts/check_entry_in_pixl_anon.sh
-
-# test export and upload
-pixl extract-radiology-reports "${PACKAGE_DIR}/test/resources/omop"
-./scripts/check_radiology_parquet.py \
-  ../exports/test-extract-uclh-omop-cdm/latest/radiology/radiology.parquet
-./scripts/check_ftps_upload.py
-
+setup () {
+  # The host OS determines how to reach the host from inside a container
+  if [ $(uname) = 'Linux' ]; then
+    export FTP_HOST=172.17.0.1
+  else
+    export FTP_HOST=host.docker.internal
```

Comment:
does using the container name work?
```suggestion
    export FTP_HOST=ftp-server
```

Reply:
ha yes of course its not